### PR TITLE
Layout fixes

### PR DESCRIPTION
**Pull request by p-szalast**

### CODE REVIEW

File: src/common/styles/componentsStyles.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import styled from "styled-components";
 import backgroundImage from "../../assets/background.png";
 
+import * as theme from "./theme";
```
Comment (cyborgx99) [inline src/common/styles/componentsStyles.ts:4]: Remind me to show you how to use Themes in Styled Components tomorrow  :)

File: src/common/styles/componentsStyles.ts [hunk 2]
```diff
@@ -11,12 +13,45 @@ export const StyledPage = styled.main`
...
   overflow: auto;
   color: #fff;
+  min-width: 1000px;
+`;
+
+export const Item = styled.div`
+  display: flex;
+  margin: 1rem;
+  padding: 1rem;
+  background: #fff;
```
Comment (cyborgx99) [inline src/common/styles/componentsStyles.ts:28]: This can also come from theme

File: src/common/styles/componentsStyles.ts [hunk 3]
```diff
@@ -11,12 +13,45 @@ export const StyledPage = styled.main`
...
+  opacity: 90%;
+  gap: 0.5rem;
+  border: 0px solid #fff;
```
Comment (cyborgx99) [inline src/common/styles/componentsStyles.ts]: does it even do anything if it's 0px?

File: src/components/OrderHistory/OrderHistoryListStyles.ts [hunk 4]
```diff
@@ -11,6 +11,6 @@ export const StyledOrderHistoryList = styled.div`
 `;
 
 export const BtnsContainer = styled(Container)`
-  margin: auto 2rem;
-  justify-content: end;
+  margin: 1rem 2rem 2rem 2rem;
+  justify-content: start;
```
Comment (cyborgx99) [inline src/components/OrderHistory/OrderHistoryListStyles.ts:15]: I'm not sure "start" does anything here. It should be at least flex-start if it's a flex container. And it's the default value anyway

File: src/components/Summary/SummaryStyles.ts [hunk 5]
```diff
@@ -1,14 +1,7 @@
 import styled from "styled-components";
 
 export const StyledSummary = styled.div`
-  margin: 1rem auto;
+  display: block;
```
Comment (cyborgx99) [inline src/components/Summary/SummaryStyles.ts]: div is block by default. No reason to indicate that

**Review by cyborgx99 — CHANGES_REQUESTED** [review]

File: src/components/OrderHistory/OrderHistoryListStyles.ts (hunk 4 above)
Comment (p-szalast) [reply src/components/OrderHistory/OrderHistoryListStyles.ts:15]: Here it extends custom component: Container and in Container justify-contnet is set to center by default

**Review by cyborgx99 — APPROVED** [review]

**Pull request merged**